arelle/PackageManager.py

```diff
@@ -83,7 +83,8 @@ def parsePackage(cntlr, filesource, metadataFile, fileBase, errors=[]):
                        file=os.path.basename(metadataFile),
                        level=logging.ERROR)
         errors.append("tpe:invalidMetaDataFile")
-        return pkg
+        if not tolerateInvalidMetadata:
+            return pkg
```

Comment:
Hi @paulwarren-wk. We discussed this as a team with @hefischer and decided that logging an error but tolerating invalid metadata should be the default behavior. Can you update this PR to remove the `tolerateInvalidMetadata` param and drop this return completely? Thank you!